source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,171 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited,
+depending on your needs : 
+
+
+Avaliable timeouts
+------------------
+
+pfSense's captive portal has 3 integrated timeout options : 
```

Comment:
Reword so that `pfSense` is not used as possessive. "The captive portal" is good enough here.
https://docs.netgate.com/pfsense/en/latest/references/style-guide.html#what-to-avoid